task6.ts

```diff
@@ -0,0 +1,11 @@
+interface Array<T> {
+    slice(start?: number, end?: number): T[];
+    pop(): T;
+    push(items: T[]): number;
```

Comment:
The type is not compatible with [JS spec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push) for `Array.push`.

![image](https://user-images.githubusercontent.com/138737/127328950-529c02a0-d9e4-4758-b0bf-fa52cbb9235d.png)